### PR TITLE
Read the docs

### PR DESCRIPTION
**Pull request by chuyskywalker**
Restores the *wickedly* cleaned up read-the-docs documentation project.

This won't render to RTD yet, but you can test it locally by making sure your `pip` is up to date and the `cd docs; make auto`. 

2016-08-06 22_38_10-welcome to pokemongo-map s documentation pokemongo-map 2 1 0 documentation

### CODE REVIEW

**Comment by Octylpy** [comment]
+1

**Comment by medghaim** [comment]
+1

**Comment by slideup-benni** [comment]
When i do "make auto" I'll get:

make: sphinx-autobuild: Kommando nicht gefunden
Makefile:233: recipe for target 'auto' failed
make: *** [auto] Error 127

What can i do?

**Comment by chuyskywalker** [comment]
gotta update them pip requirements first.

**Comment by slideup-benni** [comment]
thank you. Yes this solves all problems :)

File: docs/extras/nginx.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+   ```
+   include conf.d/pokemongo-map.conf;
+   ```
+
+3. Create a file /etc/nginx/conf.d/pokemongo-map.conf and place the following in it:
+
+   ```
+   server {
+       location /go/ {
```
Comment (hoanghuynh) [inline docs/extras/nginx.md]: @chuyskywalker I think this should be `location ^~ /go/ {` for the static assets to work.

Comment (chuyskywalker) [inline docs/extras/nginx.md]: I just ported it over, didn't actually write this doc.

**Pull request merged**